test-plans/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rust:1.65-bullseye as builder
+WORKDIR /usr/src/libp2p
+
+# TODO fix this, it breaks reproducibility
+RUN apt-get update && apt-get install -y cmake protobuf-compiler
```

Comment:
Do we still need the protobuf compiler? I thought we fixed this by inlining generated content?

Reply:
That isn't merged yet :)

Reply:
For completeness, this is tracked in https://github.com/libp2p/rust-libp2p/issues/3024.